.github/actions/run-examples/action.yml

```diff
@@ -40,11 +40,11 @@ runs:
       shell: bash -l {0}
       env:
         PIP_PROGRESS_BAR: 'off'
-        _pip_extras: '[jb,dev]'
+        _pip_extras: ''
         _pip_install_url: ${{ github.server_url}}/${{ inputs.examples-repository }}@${{ inputs.examples-ref }}
       run: |
         echo '::group::Output of `pip install idaes_examples`'
-        pip install "idaes_examples${_pip_extras} @ git+${_pip_install_url}"
+        pip install "idaes_examples${_pip_extras} @ git+${_pip_install_url}" nbmake>=1.4 jupyter-cache
```

Comment:
Is it best to "hide" these in this line? Is there a better place to put them (e.g., `requirements.txt` 😉 ) so that they are more transparent for someone who might have the same problem locally?